sklearn/tests/test_dummy.py

```diff
@@ -709,3 +709,20 @@ def test_regressor_prediction_independent_of_X(strategy):
     predictions2 = reg2.predict(X2)
 
     assert_array_equal(predictions1, predictions2)
+
+
+@pytest.mark.parametrize("strategy", [
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "strategy", ["stratified", "most_frequent", "prior", "uniform", "constant"]
)
```

If it fits on the 80 characters it would be more compact